modules/container/src/main/java/org/mule/runtime/container/internal/TestContainerModuleDiscoverer.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.container.internal;
+
+import java.util.List;
+
+/**
+ * Discovers container modules including both productive and test modules
+ *
+ * @since 4.6
+ */
+public class TestContainerModuleDiscoverer extends ContainerModuleDiscoverer {
```

Comment:
we really need the Modulediscoverer to be ecapsulated, since we won't be using it with JPMS

Reply:
Already encapsulated.